flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServer.java

```diff
@@ -202,8 +202,7 @@ public HistoryServer(
         List<RefreshLocation> refreshDirs = new ArrayList<>();
         for (String refreshDirectory : refreshDirectories.split(",")) {
             try {
-                Path refreshPath =
-                        WebMonitorUtils.validateAndNormalizeUri(new Path(refreshDirectory).toUri());
```

Comment:
This method is not unused and can be removed.

Reply:
Thanks for your review, I'll remove it.